pkg/config/config.go

```diff
@@ -7,36 +7,55 @@ import (
 	"strings"
 	"unicode"
 
-	"github.com/rs/zerolog/log"
+	"github.com/rs/zerolog"
 	flag "github.com/spf13/pflag"
 	"k8s.io/client-go/util/homedir"
 )
 
+type ZeroLogLevel int8
+
+func (l ZeroLogLevel) String() string {
+	return fmt.Sprintf("%d", l)
```

Comment:
I think we should use the zerolog level's [`.String()`](https://github.com/rs/zerolog/blob/582f0cf0e39b15ffa8d8485790063013096139df/log.go#L134) here.
```suggestion
	return zerolog.Level(l).String()
```